lib/handlers/base.rb

```diff
@@ -39,6 +39,10 @@ def initialize(settings, payload)
       @payload = payload
     end
 
+    def call
+      raise NotImplementedError
```

Comment:
https://ruby-doc.org/core-2.5.0/NotImplementedError.html

`NotImplementedError` inherits from ScriptError, which inherits from Exception. I'd avoid since it isn't a descendant of StandardError.